build.sbt

```diff
@@ -26,8 +26,9 @@ val dep = {
     "de.heikoseeberger" %% "akka-http-json4s" % "1.11.0",
     "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % "it,test",
     "io.suzaku" %% "boopickle" % "1.2.6",
-    "org.iq80.leveldb" % "leveldb" % "0.12",
-    "org.iq80.leveldb" % "leveldb-api" % "0.12",
+    "org.iq80.leveldb" % "leveldb" % "0.7",
```

Comment:
hmm i think they latest one is version `0.12` which is our own fork of leveldb with some fixes - https://github.com/input-output-hk/leveldb/releases. I know numeration does not make sense :P